csharp/PhoneNumbers/PhoneNumberUtil.cs

```diff
@@ -1200,7 +1201,107 @@ private bool HasValidCountryCallingCode(int countryCallingCode)
             return countryCallingCodeToRegionCodeMap.ContainsKey(countryCallingCode);
         }
 
+#if NET6_0_OR_GREATER
         /// <summary>
+        /// Formats a phone number in the specified format using default rules. Note that this does not
+        /// promise to produce a phone number that the user can dial from where they are - although we do
+        /// format in either 'national' or 'international' format depending on what the client asks for, we
+        /// do not currently support a more abbreviated format, such as for users in the same "area" who
+        /// could potentially dial the number without area code. Note that if the phone number has a
+        /// country calling code of 0 or an otherwise invalid country calling code, we cannot work out
+        /// which formatting rules to apply so we return the national significant number with no formatting
+        /// applied.
+        /// </summary>
+        /// <param name="number">The phone number to be formatted.</param>
+        /// <param name="numberFormat">The format the phone number should be formatted into.</param>
+        /// <returns>The formatted phone number.</returns>
+        public string Format(PhoneNumber number, PhoneNumberFormat numberFormat)
+        {

```

Comment:
As to not be breaking `public void Format(PhoneNumber number, PhoneNumberFormat numberFormat, StringBuilder formattedNumber)` should still exists.

Reply:
yeah, you are right, i forgot to move it outside `#else` statement. i'll fix it in the next commit

Reply:
should be fine now